bin/fixtures/_test.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "Empty fixture to run as a test"
+}
```

Comment:
This is an empty fixture that the tests run to make sure that nothing fails in a simple benchmark run. It's excluded when running `all`.

Reply:
hah i had meant to do this a while back too, this is great. much faster feedback when testing changes to the benchmark suites themselves